sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -58,23 +58,25 @@ az cognitiveservices account create \
     --yes
 ```
 ### Authenticate the client
-In order to interact with the Form Recognizer service, you will need to create an instance of the `FormRecognizerClient` 
-class. You will need an **endpoint** and an **key** to instantiate a client object, 
+In order to interact with the Form Recognizer service, you will need to create an instance of the Form Recognizer client,
+both the asynchronous and synchronous clients can be created by using `FormRecognizerClientBuilder` invoking `buildClient()`
+creates a synchronous client while `buildAsyncClient` creates its asynchronous counterpart. 
+
+You will need an **endpoint** and a **key** to instantiate a client object, 
 they can be found in the [Azure Portal][azure_portal] under the "Quickstart" in your created
 Form Recognizer resource. See the full details regarding [authentication][authentication] of Cognitive Services.
 
-#### Get credentials
-The `credential` parameter may be provided as a [`AzureKeyCredential`][azure_key_credential] from [azure-core][azure_core].
-
-##### Create FormRecognizerClient with AzureKeyCredential
-To use AzureKeyCredential authentication, provide the [key][key] as a string to the [AzureKeyCredential][azure_key_credential]. This can be found in the [Azure Portal][azure_portal] 
-   under the "Quickstart" section or by running the following Azure CLI command:
+#### Create a Form Recognizer client using AzureKeyCredential
+To use `AzureKeyCredential` authentication, provide the [key][key] as a string to the [AzureKeyCredential][azure_key_credential]. 
+This key can be found in the [Azure Portal][azure_portal] under the "Quickstart" section in your created Form Recognizer
```

Comment:
The endpoint and key don't seem to be found under "Quickstart" anymore. You have to click on "Keys and Endpoint".